src/Entry/Exporter/Exporter.cpp

```diff
@@ -126,7 +126,7 @@ class Exporter__pImpl
 {
   Exporter* q_api; // NOLINT
 public:
-  std::shared_ptr<layer::GraphicsContext> ctx;
+  std::shared_ptr<VkGraphicsContext> ctx;
```

Comment:
This completely disables the possibility of EGL. Is this appropriate?

Reply:
This is nothing about EGL or any other backends. It's just a type of variable.